REFACTOR.md

```diff
@@ -0,0 +1,18 @@
+# Refactoring Baikal to the Silex framework
+
+- Original webroot is now at 'html.old'
+- New webroot remains the same 'html' for easier upgrades
+- Composer autoloader switch to psr-4 and support both locations
```

Comment:
One of the biggest reasons for me making the switch was that it was indeed a small composer change, but also to be able to have both old and new style to keep working alongside each other. Therefore being able to compare while running two PHP web servers simultaneously during development.